src/beam.ts

```diff
@@ -675,19 +732,19 @@ export class Beam extends Element {
   }
 
   // Get the x coordinates for the beam lines of specific `duration`
-  getBeamLines(duration) {
+  getBeamLines(duration: string): BeamInfo[] {
```

Comment:
No need for separate exported type `BeamInfo` here, okay to return `{ start: number, end?: number }[]`

Reply:
If I define the type, couldn´t I use it here as well?

Reply:
ok you wanted the type local